front-end/src/app/shared/resolvers/transaction-type.resolver.ts

```diff
@@ -64,6 +68,12 @@ export class TransactionTypeResolver implements Resolve<TransactionType | undefi
             transaction.transaction_type_identifier
           ) as TransactionType;
           transactionType.transaction = transaction;
+
+          if (transactionType.childTransactionType) {
+            transactionType.childTransactionType.transaction = undefined; // TODO: Need to populate this with what is coming back from the API, ideally it is sending a list of 2 txns
```

Comment:
https://github.com/fecgov/fecfile-web-api/pull/233
this is going to need to be a subsequent call using the parent_transaction_id filter

Reply:
code updated so that parent transaction is in payload